pkg/config/config.go

```diff
@@ -212,7 +212,7 @@ func init() {
 	Datadog.SetDefault("kube_resources_namespace", "")
 
 	// Datadog cluster agent
-	Datadog.SetDefault("cluster_agent", false)
+	Datadog.SetDefault("cluster_agent.enabled", false)
 	Datadog.SetDefault("cluster_agent.auth_token", "")
 	Datadog.SetDefault("cluster_agent.url", "")
 	Datadog.SetDefault("cluster_agent.kubernetes_service_name", "dca")
```

Comment:
We also discussed changing the default service name from `dca` to `datadog-cluster-agent`. This would require updates to other docs and the sample manifests.